tests/util/types.js

```diff
@@ -0,0 +1,135 @@
+const types = {
+	any: {
+		syntax: "*",
+		initialValue: "0",
+		values: ["foo", "42"],
```

Comment:
```suggestion
		values: ["foo", "42"],
		property: "--foo",
```

Reply:
We should have a property to test this type, right?

Reply:
`property` is for built-ins, we generate a custom property for all of them anyway.